tests/buildkite/build-rpkg-win64-gpu.ps1

```diff
@@ -5,12 +5,12 @@ $ErrorActionPreference = "Stop"
 Write-Host "--- Build XGBoost R package with CUDA"
 
 nvcc --version
-if ( $is_release_branch -eq 1 ) {
-  $arch_flag = "-DGPU_COMPUTE_VER=75"
+$arch_flag = "-DGPU_COMPUTE_VER=75"
```

Comment:
This line is actually not used. In a follow-up PR, I will find ways to only build `sm_75` to speed up builds for pull requests. (For `master`, still need to build all `sm`s.)